### ISSUE TITLE
Unread count doesn't update in root window

### ISSUE THREAD

**Issue by TatriX** [zevlg/telega.el#81]
I have this window setup:
```
;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
;; telega-root | telega-chat with user x ;;
;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;

```
Focus left in the chat window. Then when I'm getting a message from the user `x`, and switch focus back to Emacs with that chat focused, I assume that the message should be marked as read. But telega-root window and modeline will still show me counter for unread messages for user `x`. And the only way to get rid of it is to switch back to root window and press RET on that chat.

**Comment by zevlg** [comment]
Yeah, telega at the moment does not track X11 focus migration.  There is however method to do so in Emacs, e.g. `after-focus-change-function`, telega might utilize it to implement this feature

thanks

**Comment by zevlg** [comment]
Fixed in 0.4.3, please update
